requirements/static/py3.7/windows.txt

```diff
@@ -79,10 +79,11 @@ pymysql==0.9.3
 pyopenssl==19.0.0
 pyparsing==2.4.5          # via packaging
 pytest-helpers-namespace==2019.1.8
-pytest-salt-runtests-bridge==2019.7.10
-pytest-salt==2019.12.18
+pytest-salt-runtests-bridge==2019.12.5
```

Comment:
I'm sure there's a reasonable reason that we're using `pytest-salt-runtests-bridge` even though we've removed all the runtests stuff(??). Could we have an explanation for those of us unfamiliar with it?

Reply:
It provides test listing from files and test whitelist from a file compatability layer.
Eventually we might drop it.